MAINTAINERS.md

```diff
@@ -17,6 +17,7 @@ libuv is currently managed by the following individuals:
   - GPG key: 9DFE AA5F 481B BF77 2D90  03CE D592 4925 2F8E C41A (pubkey-iwuzhere)
 * **Jameson Nash** ([@vtjnash](https://github.com/vtjnash))
   - GPG key: AEAD 0A4B 6867 6775 1A0E  4AEF 34A2 5FB1 2824 6514 (pubkey-vtjnash)
+  - GPG key: CFBB 9CA9 A5BE AFD7 0E2B  3C5A 79A6 7C55 A367 9C8B (pubkey2022-vtjnash)
```

Comment:
`pubkey2022-vtjnash` doesn't appear to exist in neither this repository 
```
git ls-remote https://github.com/libuv/libuv | grep pubkey
```
nor 
```
git ls-remote https://github.com/vtjnash/libuv | grep pubkey
```
were you waiting for this PR to be merged before pushing the key?

Reply:
Not intentionally waiting, no. But now should exist beyond my local computer haha